packages/loader/container-loader/src/catchUpMonitor.ts

```diff
@@ -3,34 +3,29 @@
  * Licensed under the MIT License.
  */
 
-import { IDisposable, IEvent } from "@fluidframework/common-definitions";
-import { assert, TypedEventEmitter } from "@fluidframework/common-utils";
+import { IDisposable } from "@fluidframework/common-definitions";
+import { assert } from "@fluidframework/common-utils";
 import { IDeltaManager } from "@fluidframework/container-definitions";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
 
-/** @see ICatchUpMonitor for usage */
+/** @see CatchUpMonitor for usage */
 type CaughtUpListener = () => void;
 
-/** @see ICatchUpMonitor for usage */
-export interface ICatchUpMonitorEvents extends IEvent {
-    (event: "caughtUp", listener: CaughtUpListener): void;
-}
-
 /** Monitor that emits an event when a Container has caught up to a given point in the op stream */
-export interface ICatchUpMonitor extends TypedEventEmitter<ICatchUpMonitorEvents>, IDisposable { }
+export type ICatchUpMonitor = IDisposable;
```

Comment:
Just implement IDisposable below?

Reply:
I can change it - I left it to reduce number of changes, and leave it open for possible future additions (i.e. maybe we will expose more state on this object in the future).
But happy to remove type if you feel like it - I do not care that much :)

Reply:
Makes sense, fine either way.